R/hazardFunctions.R

```diff
@@ -20,23 +20,46 @@ ExpHazOS <- function(t, h01, h02, h12) {
   ((h12 - h02) * (h01 + h02) - h01 * h12 * exp(-h012 * t)) / ((h12 - h02) - h01 * exp(-h012 * t))
 }
 
+#' Helper Function for `avgHRExpOS()`
+#'
```

Comment:
Can you add a description what this is calculating? e.g. "Integrand of the form ... " to get an idea what this is doing